addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -720,6 +720,10 @@ pre {
                     border-color: $-btn-primary-color;
                 }
             }
+
+            .btn-close {
+                filter: #{if(lightness($-bg-color) < 50, 'var(--btn-close-white-filter)', 'none')};
+            }
```

Comment:
Initially I wanted to suggest to use `unset` rather than `none` in order to avoid enforcing "a value" when the condition is not meet. Still, even `unset` is "a value", and the rule may override something else (unlikely... but still).

I think that to add the rule only when necessary is safer ;)

```suggestion
            .btn-close {
                @if(lightness($-bg-color) < 50) {
                    filter: var(--btn-close-white-filter);
                }
            }
```

Reply:
I'll use this system in `website.scss` but I'll have to keep what I did here because we need to override the filter if my website's `body-bg` appears to be dark AND we are using an `o_cc` with a bright color. I'll use `unset` instead of `none` though

<img width="1656" height="167" alt="Image" src="https://github.com/user-attachments/assets/c30bd860-a5f0-4cf9-a839-fbe75d7659fa" />